specials/SPC30_Matias_IslandInfiltration.Malden/do_not_edit/initServer.sqf

```diff
@@ -0,0 +1,27 @@
+// Calculate mission start
+
+systemTimeUTC params ["", "", "", "_hour", "_minute"];
```

Comment:
Can remove everything from this file since you're going to have the players pick the time

Reply:
I've deleted it and stopped it from being included in the initServer from the main folder